src/login/use-action-button.js

```diff
@@ -0,0 +1,13 @@
+import { useGoogleLogin, useGoogleLogout } from 'react-google-login'
+
+import { useContext } from 'login/context'
+import { useConfigLogout } from 'logout/logout-config'
+import { useConfig } from 'login/login-config'
+
+export const useButtonAction = () => {
+  const { signIn } = useGoogleLogin(useConfig())
+  const { signOut } = useGoogleLogout(useConfigLogout())
+  const { user } = useContext()
+
+  return user ? signOut : signIn
+}
```

Comment:
Whould the name `useButtonCallback` be more relevant here?

Reply:
Should we also rename the file into `use-button-callback.js`?

Reply:
Done